woocommerce/woocommerce-ios
# Refunds Step 2: Storage - Mark I

Partial work for #1096.

⚠️  Dependent upon `1307-refunds-network-ii` branch. Merge that first.

This PR adds the `OrderRefundCondensed` entity to the data model and adds the `Order.refunds` relationship.

### To test
This will break CircleCI's unit tests, due to the Order Networking model getting altered in PR #1311. 
To test: in Xcode, set the target to the Storage framework. Run the unit tests for Storage. All Storage unit tests must pass.
CircleCI's unit tests should all pass when Yosemite is edited, in PR #1313.

Update release notes:
